### DISCUSSION TITLE
Need help with LLCC68 debugging

### DISCUSSION THREAD

**Discussion by maalos** [jgromes/RadioLib#1352] [Q&A] [answered]
So, I'm trying to "port" my LLCC68 (reported as SX1262) to https://github.com/sh123/esp32_loradv and got over nearly all issues but one - RADIOLIB_ERR_TX_TIMEOUT, despite having DIO1 connected to the configured pin. I've checked and replaced cables, but I just cannot get it to work. Any ideas on how I could debug it? The module doesn't seem transmit anything either, checked it with my handheld radio on monitor mode.
![image](https://github.com/user-attachments/assets/dd348c9b-bdce-4e63-8fb5-2a0cbec4ea22)
![image](https://github.com/user-attachments/assets/c885ffb6-6c2f-4c05-924a-6246122d0873)

**Comment by jgromes** [comment]
Couple of things can be going on here. The strange thing is that the radio seemingly never starts transmitting - can you verify there is something to actually transmit?

**Reply by maalos** [reply]
> can you verify there is something to actually transmit?

I've connected the INMP441 microphone and can see data flowing through the circular buffer. It's now transmitting, although it doesn't sound like LoRa whatsoever. Anyways, the error is still there despite transmitting data
![image](https://github.com/user-attachments/assets/2481404e-2f51-47d2-8ced-b333b289fa7a)

**Reply by jgromes** [reply]
My first step would be to try and isolate the issue. Does this laso happen in a simple setup, i.e. just transmitting a simple string? Does it depend on the size of the data to be sent? Or some LoRa setting?

**Reply by maalos** [reply, marked as answer]
I've ditched that code as it was too complicated for my needs, I didn't figure out the issue. Thank you, the discussion can be closed now.
